Sources/SwiftRefactor/ConvertComputedPropertyToZeroParameterFunction.swift

```diff
@@ -0,0 +1,101 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#if swift(>=6)
+public import SwiftSyntax
+#else
+import SwiftSyntax
+#endif
+
+public struct ConvertComputedPropertyToZeroParameterFunction: SyntaxRefactoringProvider {
+  public static func refactor(syntax: VariableDeclSyntax, in context: Void) -> FunctionDeclSyntax? {
+    guard syntax.bindings.count == 1,
+      let binding = syntax.bindings.first,
+      let identifierPattern = binding.pattern.as(IdentifierPatternSyntax.self)
+    else { return nil }
+
+    var statements: CodeBlockItemListSyntax
+
+    guard let typeAnnotation = binding.typeAnnotation,
+      let accessorBlock = binding.accessorBlock
+    else { return nil }
+
+    switch accessorBlock.accessors {
+    case .accessors(let accessors):
+      guard accessors.count == 1, let accessor = accessors.first,
+        accessor.accessorSpecifier.tokenKind == .keyword(.get), let codeBlock = accessor.body
+      else { return nil }
+      statements = codeBlock.statements
+      statements.leadingTrivia = accessor.leadingTrivia + codeBlock.leadingTrivia + statements.leadingTrivia
+      statements.trailingTrivia += codeBlock.trailingTrivia + accessor.trailingTrivia
+      statements.trailingTrivia = statements.trailingTrivia.droppingTrailingWhitespace
```

Comment:
We are still dropping the trailing trivia here (eg. if you have `get /* comment */ {`). I would suggest that we add the leading trivia of the tokens here.

```suggestion
      statements.leadingTrivia = accessor.accessorSpecifier.trivia + codeBlock.leftBrace.trivia + statements.leadingTrivia
      statements.trailingTrivia += codeBlock.rightBrace.trivia
      statements.trailingTrivia = statements.trailingTrivia.droppingTrailingWhitespace
```